rules/falco_rules.yaml

```diff
@@ -2395,6 +2393,20 @@
     - registry.ng.bluemix.net/armada-master/metrics-server-amd64
     - registry.ng.bluemix.net/armada-master/olm
 
+# Falco containers
+- list: falco_containers
```

Comment:
Can we move the list declaration on top before to actually use it. this would raise some errors and it would be better to find the macro if declared before to use it.

Reply:
No problem! I moved it before the first usage.

Reply:
@oscr I still see the list where it was. Do you mind do a double check pls?

Reply:
@darryk10 I'm sorry! I forgot to actually add the file before amending and pushing :sweat_smile: Fixed now